omero/sysadmins/limitations.rst

```diff
@@ -79,6 +79,23 @@ This primarily affects the following file formats:
 This issue can be avoided by pre-generating pyramidal OME-TIFF images as
 described above.
 
+.. _ngff_limitations:
+
+Import of OME-NGFF
+^^^^^^^^^^^^^^^^^^
+
+The import of `OME-NGFF <https://ngff.openmicroscopy.org/latest/>`_ is currently limited to the :doc:`command-line (CLI) importer </users/cli/import>` only.
+
+Naming of OME-NGFF images in OMERO
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default naming of the `OME-NGFF <https://ngff.openmicroscopy.org/latest/>`_ Images imported into OMERO is not intuitive at the moment. Use the ``-n NAME`` option of the :doc:`command-line (CLI) importer </users/cli/import>` to achieve explicit naming.
+
+Depth of scanning prior to import
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The import might fail if the hierarchy of folders is exceeding the depth of scanning (default: 4). For formats using deeper hierarchy of folders such as  `OME-NGFF <https://ngff.openmicroscopy.org/latest/>`_ use ``--depth 10`` to set the depth of scanning of 10 (or more if necessary) as described in the :doc:`command-line (CLI) importer </users/cli/import>`.
```

Comment:
Same as above, I assumption we can use the option markup to link directly to `omero import --depth`